web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -796,27 +796,18 @@ class AppVersionHistory extends Component<Props, State> {
     }
   };
 
-  redeployVersion = (version: Version, isRollback = false) => {
```

Comment:
this is never called with `true` as a second argument so I'm just getting rid of it. The whole rollback situation is pretty messy right now so we'll probably clean that up once I get the hooks integrated